dependencyManagement/build.gradle.kts

```diff
@@ -88,16 +84,16 @@ val DEPENDENCIES = listOf(
   "io.opentracing:opentracing-api:0.33.0",
   "io.zipkin.zipkin2:zipkin-junit:2.23.2",
   "junit:junit:4.13.2",
```

Comment:
Out of curiosity - why do we have two junit versions ("org.junit:junit-bom:5.7.2" as BOM for the second one)?

Reply:
We have some test helpers for junit 4 as well as using junit 5 ourselves for everything.